toolkit/tools/imagecustomizer/main.go

```diff
@@ -38,14 +40,16 @@ type InjectFilesCmd struct {
 }
 
 type RootCmd struct {
-	Customize     CustomizeCmd     `name:"customize" cmd:"" default:"withargs" help:"Customizes a pre-built Azure Linux image."`
-	InjectFiles   InjectFilesCmd   `name:"inject-files" cmd:"" help:"Injects files into a partition based on an inject-files.yaml file."`
-	Version       kong.VersionFlag `name:"version" help:"Print version information and quit"`
-	TimeStampFile string           `name:"timestamp-file" help:"File that stores timestamps for this program."`
+	Customize        CustomizeCmd     `name:"customize" cmd:"" default:"withargs" help:"Customizes a pre-built Azure Linux image."`
+	InjectFiles      InjectFilesCmd   `name:"inject-files" cmd:"" help:"Injects files into a partition based on an inject-files.yaml file."`
+	Version          kong.VersionFlag `name:"version" help:"Print version information and quit"`
+	TimeStampFile    string           `name:"timestamp-file" help:"File that stores timestamps for this program."`
+	DisableTelemetry bool             `name:"disable-telemetry" help:"Disable telemetry collection of the tool."`
```

Comment:
Considering that `OTEL_EXPORTER_OTLP_ENDPOINT` has to be specified, we probably don't need `--disable-telemetry` in the exe now.